PDOSetup.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+function setUpPDO () {
```

Comment:
isnt being used everywhere that PDO is being set up, either delete or use everywhere